openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -125,3 +121,7 @@
 <%static:require_module_async module_name="js/dateutil_factory" class_name="DateUtilFactory">
     DateUtilFactory.transform('.localized-datetime');
 </%static:require_module_async>
+
+<%static:webpack entry="CourseOutline">
+    new CourseOutline();
```

Comment:
Since the rendering is done all on the server, the JS run inside the factory doesn't actually need to know anything about the container for the tree UI, so I removed it - but I agree the pattern you're describing is the one we'll want for future es2015 module/factories that do render UI (e.g., react ones). Is it ok if we wait to add the container class name as an argument to the constructor back until we move this UI into React?